package.json

```diff
@@ -43,7 +43,7 @@
     "eslint-config-react-app": "^7.0.1",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "prettier": "2.8.8"
+    "prettier": "^3.0.0"
```

Comment:
I think it's better to point to the exact version for prettier (that's what the official doc suggests as well), as automatic upgrades of prettier might cause unexpected lint errors at pull reqs. GitHub bot can automate upgrades of it for us.